account_fiscal_year/models/res_company.py

```diff
@@ -0,0 +1,83 @@
+#  Copyright 2020 Simone Rubino - Agile Business Group
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from datetime import timedelta
+
+from odoo import models
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT, date_utils
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    def compute_fiscalyear_dates(self, current_date):
```

Comment:
This method has been recently re-added (https://github.com/odoo/odoo/commit/544cb3682033e9f644b48b97dd5197eaf9282ff5), thus tests can be fail when both `account_fiscal_year` and `account_accounting` (enterprise module) are installed: I suggest you then to rename it. Thanks!

Reply:
Nice catch! I've been thinking a lot about this.
First, the fallback method in Odoo CE makes little sense to me.
Second, the fiscal year exists outside Odoo too, so maybe there's one official way to compute it? If that's the case then EE and OCA's way should be aligned already.

**To rename**
If I rename the method (`cfyd` for short), say to `cfyd_OCA`, there will be two separate ways (read: `methods`) to compute the fiscal year: one that follows the _OCA way_ and one that follows the _Odoo CE/EE way_.
I think this can be misleading and result in unexpected behavior: 

- some modules might be calling `cfyd` and get something when they have only CE installed, then it changes when installing EE, and installing this module has no effect; 
- some modules might be calling `cfyd_OCA` but then the result does not match with what is computed by CE and potentially doesn't even match with what EE computes.

**Not to rename**
Not renaming it, on the other hand, will cause tests to fail but as I said if there's one day to do it, then everyone should be getting the same result.
Moreover at least there will be only 'one way to do it'.

**?**
There might be a third approach but I can't see it.

@pedrobaeza I think I'll use the help you kindly offered in https://github.com/OCA/account-financial-tools/issues/1063#issuecomment-722453974, here there is one big doubt I have :)

Reply:
@pedrobaeza do you have any idea about this?
Maybe this kind of issue has already happened and there is a 'standard way' to deal with it?

Reply:
I would go for "**Not to rename**": `compute_fiscalyear_dates`  method should return the same result both in OCA and Enterprise

Reply:
@tafaRU I'd rather not rename it for the reasons above, is it ok for you?

Reply:
It's fine by me :+1: